### PR TITLE
Node.js bindings: Remove sol_oic_server_(init|shutdown)

### PR DESCRIPTION
**Pull request by gabrielschulhof**
Signed-off-by: Gabriel Schulhof <gabriel.schulhof@intel.com>

### CODE REVIEW

File: bindings/nodejs/tests/tests/OIC Server.js [hunk 1]
```diff
@@ -22,10 +22,7 @@ var testUtils = require( "../assert-to-console" );
 var theResource;
 var anError = {};
 
-console.log( JSON.stringify( { assertionCount: 4 } ) );
-
-testUtils.assert( "strictEqual", soletta.sol_oic_server_init(), 0,
-	"sol_oic_server_init() is successful" );
+console.log( JSON.stringify( { assertionCount: 3 } ) );
```
Comment (glima) [inline bindings/nodejs/tests/tests/OIC Server.js]: It seems the work on ca5d3a9a0e9d8145bb0b44a540985c9fd2d0ddef missed this. Okay by me.

Comment (otaviobp) [inline bindings/nodejs/tests/tests/OIC Server.js]: Sorry about that. I'll be sure to check Node.js bindings when I change any API in oic again.
Thanks
+1 for commti

Comment (glima) [inline bindings/nodejs/tests/tests/OIC Server.js]: It's more a matter of checking the bot status for each commit :)

**Comment by cabelitos** [comment]
merged

**Pull request closed without merging**